src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CreateSomaticPanelOfNormals.java

```diff
@@ -0,0 +1,151 @@
+ */
+@CommandLineProgramProperties(
+        summary = "Make a somatic panel of normals",
+        oneLineSummary = "Make a somatic panel of normals",
+        programGroup = VariantProgramGroup.class
+)
+public class CreateSomaticPanelOfNormals extends CommandLineProgram {
+
+    public static final String INPUT_VCFS_LIST_LONG_NAME = "vcfsListFile";
+    public static final String INPUT_VCFS_LIST_SHORT_NAME = "vcfs";
+
+    /**
+     * Plain text file listing one normal vcf per line. This argument can be specified
+     * multiple times in order to provide multiple vcf lists.
+     */
+    @Argument(fullName = INPUT_VCFS_LIST_LONG_NAME,
+            shortName = INPUT_VCFS_LIST_SHORT_NAME,
+            doc="File containing a list of samples to include", optional = false)
+    private Set<File> vcfLists = new LinkedHashSet<>(0);
+
+    @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="Output vcf", optional = false)
+    private File outputVcf = null;
+
+    public Object doWork() {
+        final List<File> inputVcfs = getInputVcfs(vcfLists);
+        final Collection<CloseableIterator<VariantContext>> iterators = new ArrayList<>(inputVcfs.size());
+        final Collection<VCFHeader> headers = new HashSet<>(inputVcfs.size());
+        final VCFHeader headerOfFirstVcf = new VCFFileReader(inputVcfs.get(0), false).getFileHeader();
+        final SAMSequenceDictionary sequenceDictionary = headerOfFirstVcf.getSequenceDictionary();
+        final VariantContextComparator comparator = headerOfFirstVcf.getVCFRecordComparator();
+
+
+        for (final File vcf : inputVcfs) {
+            final VCFFileReader reader = new VCFFileReader(vcf, false);
+            iterators.add(reader.iterator());
+            final VCFHeader header = reader.getFileHeader();
+            Utils.validateArg(comparator.isCompatible(header.getContigLines()), () -> vcf.getAbsolutePath() + " has incompatible contigs.");
+            headers.add(header);
+        }
+
+        final VariantContextWriter writer = GATKVariantContextUtils.createVCFWriter(outputVcf, sequenceDictionary, false, Options.INDEX_ON_THE_FLY);
+        writer.writeHeader(new VCFHeader(VCFUtils.smartMergeHeaders(headers, false)));
+
+        final MergingIterator<VariantContext> mergingIterator = new MergingIterator<>(comparator, iterators);
+        SimpleInterval currentPosition = new SimpleInterval("FAKE", 1, 1);
+        final List<VariantContext> variantsAtThisPosition = new ArrayList<>(20);
+        while (mergingIterator.hasNext()) {
+            final VariantContext vc = mergingIterator.next();
+            if (!currentPosition.overlaps(vc)) {
+                processVariantsAtSamePosition(variantsAtThisPosition, writer);
+                variantsAtThisPosition.clear();
+                currentPosition = new SimpleInterval(vc.getContig(), vc.getStart(), vc.getStart());
```

Comment:
I'm using `new SimpleInterval(vc.getContig(), vc.getStart(), vc.getStart())` and not `new SimpleInterval(vc.getContig(), vc.getStart(), vc.getEnd())` as an attempt to replicate the old behavior where we look at all variants with a common start.